plugin/src/test/groovy/com/novoda/staticanalysis/internal/checkstyle/CheckstyleIntegrationTest.groovy

```diff
@@ -44,6 +44,29 @@ public class CheckstyleIntegrationTest {
     }
 
     @Test
+    public void shouldFailBuildAgainWhenCheckstyleWarningsStillOverTheThresholdAfterSecondRun() {
```

Comment:
👍 

might be nice to word it differently tho

`afterSecondRunWhenStillOverTheThresholdThenTheBuildFailsAgain`

or if you insist on starting with should:

`shouldFailTheBuildAfterSecondRunWhenStillOverTheThreshold`